Models/getModels.js

```diff
@@ -10,5 +10,5 @@ exports.selectCategories = () => {
     .then((results) => {
       return results.rows;
     })
-    .catch(console.log);
+    .catch(next);
```

Comment:
This catch would be more suited to being in the controller is a very minor comment.